test/fixtures/compile-test-data.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
(the test .js files were woefully out of date, and dust doesn't guarantee compatibility if you use a different compiler and runtime version)